scripts/test/nightly/runTests.sh

```diff
@@ -108,12 +143,53 @@ runMavenSystemTests() {
     echo -n "$TEST_EXIT_CODE $SHORT_ID" > "$OUTPUT_DIR/$TEST_NAME.status"
 }
 
-runMavenSystemTests "${DEPLOY_ID}mvn${START_TIME_SHORT}" $MAIN_SUITE_NAME "${MAIN_SUITE_PARAMS[@]}"
+runTestSuite(){
+    SUITE=$3
+    ##copyFolderForParallelRun "$SUITE"
+    sleep $1 #Delay so that initial deployment doesn't clash with each other
+    shift 1
+    echo "[$(time_str)] Starting test suite: $SUITE"
+    pushd "$REPO_PARENT_DIR/sleeper/$SUITE/scripts/test" #Move into isolated repo copy
+    runMavenSystemTests "$@"
+    popd
+    echo "[$(time_str)] Finished test suite: $SUITE"
+    removeFolderAfterParallelRun "$SUITE"
+}
+
+runSlowTests(){
+    runTestSuite 0  "${DEPLOY_ID}${START_TIME_SHORT}q1" "quick" "-DskipRust" "-DrunIT=SlowSuite1" "$@" ##&
+    ##runTestSuite 60 "${DEPLOY_ID}${START_TIME_SHORT}s1" "slow1" "-DskipRust" "-DrunIT=SlowSuite1" "$@" &
+    ##runTestSuite 120 "${DEPLOY_ID}${START_TIME_SHORT}s2" "slow2" "-DskipRust" "-DrunIT=SlowSuite2" "$@" &
+    ##runTestSuite 180 "${DEPLOY_ID}${START_TIME_SHORT}s3" "slow3" "-DskipRust" "-DrunIT=SlowSuite3" "$@"
+}
+
+if [ "$MAIN_SUITE_NAME" == "performance" ]; then
+    echo "Running performance tests in parallel. Start time: [$(time_str)]"
+    SUITE_PARAMS=(-Dsleeper.system.test.cluster.enabled=true -DskipRust)
+    EXP1_SUITE_PARAMS=("${DEPLOY_ID}${START_TIME_SHORT}e1" "expensive1" "${SUITE_PARAMS[@]}" -DrunIT=ExpensiveSuite1)
+    EXP2_SUITE_PARAMS=("${DEPLOY_ID}${START_TIME_SHORT}e2" "expensive2" "${SUITE_PARAMS[@]}" -DrunIT=ExpensiveSuite2)
+    EXP3_SUITE_PARAMS=("${DEPLOY_ID}${START_TIME_SHORT}e3" "expensive3" "${SUITE_PARAMS[@]}" -DrunIT=ExpensiveSuite3)
+
+    runSlowTests "$@" &
+    runTestSuite 240 "${EXP1_SUITE_PARAMS[@]}" "$@" &
+    runTestSuite 300 "${EXP2_SUITE_PARAMS[@]}" "$@" &
+    runTestSuite 360 "${EXP3_SUITE_PARAMS[@]}" "$@"
+    wait
+elif [ "$MAIN_SUITE_NAME" == "functional" ]; then
+    echo "Running slow tests in parallel. Start time: [$(time_str)]"
+    runSlowTests $@
```

Comment:
This `$@` needs quoting as in the previous comment https://github.com/gchq/sleeper/pull/5762#discussion_r2447277916.